src/components/Topbar/StudioTopbar/StudioTopbar.style.tsx

```diff
@@ -59,18 +59,11 @@ export const TextContainer = styled.div`
   }
 `
 
-export const StudioContainer = styled.div`
+export const StudioTopbarContainer = styled.div`
   display: flex;
   align-items: center;
   margin-left: auto;
   margin-right: ${sizes(3)};
-  svg {
-    width: auto;
-    height: auto;
-  }
-  button {
-    cursor: pointer;
```

Comment:
Removing this makes the `pointer` go away. Can you look into the `Button` component and see why it doesn't apply this style?